tools/chef/cookbooks/px_dev/recipes/linters.rb

```diff
@@ -26,7 +26,7 @@
 end
 
 execute 'install py linters' do
-  command 'python3 -m pip install flake8 flake8-mypy yamllint --no-cache-dir && python3 -m pip cache purge'
+  command 'python3 -m pip install --break-system-packages flake8 flake8-mypy yamllint --no-cache-dir && python3 -m pip cache purge'
```

Comment:
What system package does this install conflict with? can we side step this by doing a user install?
This also seems ok, but the flag naming usually alarms folks. :)

Reply:
I don't think these packages actually conflict with anything currently. I believe Ubuntu is marking its python as externally managed according to [PEP 668](https://peps.python.org/pep-0668/) and so any system wide pip install is going to complain with Ubuntu's python.

My understanding is that we would need to install the ubuntu apt versions of these dependencies:
* [yamllint](https://packages.ubuntu.com/en/noble/yamllint)
* [flake8](https://packages.ubuntu.com/en/noble/flake8)
* flake8-mypy [upstream](https://github.com/ambv/flake8-mypy) is deprecated so that should probably be replaced

What if we replaced flake8 and yamllint with Ubuntu's packages and deferred "fixing" flake8-mypy until it is replaced with a different linter integration?